command/cp.go

```diff
@@ -956,3 +998,57 @@ func guessContentType(file *os.File) string {
 	}
 	return contentType
 }
+
+type CustomWriter struct {
+	c  Copy
+	fp *os.File
+}
```

Comment:
We do not need a reference to the cp command, progress bar is enough.
I didn't like the name `Custom` :) it is too generic. Let's be precise about what this writer does. Here is my suggestion:

```suggestion
type CountingWriter struct {
	pb  progressbar.ProgressBar
	fp *os.File
}
```

Reply:
Yes, I didn't like it either but I couldn't find another name. Thanks for the suggestion.